tests/utils/test_databricks_utils.py

```diff
@@ -105,12 +106,10 @@ def test_get_workspace_info_from_databricks_secrets():
 
 def test_get_workspace_info_from_dbutils():
     mock_dbutils = mock.MagicMock()
-    mock_dbutils.notebook.entry_point.getDbutils.return_value.notebook.return_value.getContext.return_value.browserHostName.return_value.get.return_value = (  # noqa
-        "mlflow.databricks.com"
-    )
-    mock_dbutils.notebook.entry_point.getDbutils.return_value.notebook.return_value.getContext.return_value.workspaceId.return_value.get.return_value = (  # noqa
-        "1111"
-    )
+    methods = ["notebook.entry_point.getDbutils", "notebook", "getContext"]
+    mock_method_chain(mock_dbutils, methods + ["browserHostName", "get"], "mlflow.databricks.com")
+    mock_method_chain(mock_dbutils, methods + ["workspaceId", "get"], "1111")
```

Comment:
nit: It would be nice to put the last argument as a keyword argument: `return_value="mlflow.databricks.com"` and `return_value="1111"` for readability.
Same for the rest of this file.

Reply:
Makes sense, I'll fix the code to use keyword arguments :)